### ISSUE TITLE
2D camera support

### ISSUE THREAD

**Issue by marshauf** [djeedai/bevy_hanabi#17]
Hello,
is it possible to use bevy_hanabi with a 2D camera, like OrthographicCameraBundle::new_2d() ? If so, how can I use it?

**Comment by djeedai** [comment]
No unfortunately there's no simple way to do that. This has to do with the fact that Bevy completely separate 2D and 3D rendering, and so I had to choose to integrate into one of those paths. This is really a shame, but this is a widespread issue with all features implemented only for 1 of the 2 codepaths. We would have to provide a second render path specifically for 2D to make that work, and at the minute this is not a priority given most other things are under heavy development / still unstable. If Bevy continues down this path though I might consider supporting both 2D and 3D in parallel but this is not trivial.

**Comment by Sheepyhead** [comment]
I'm just adding that I would also love to have a 2d version of this crate :smile:

**Comment by djeedai** [comment]
@josh65536 added support via #18, closing this.
